core/src/main/java/org/apache/accumulo/core/client/impl/TableOperationsImpl.java

```diff
@@ -244,7 +244,7 @@ private long beginFateOperation() throws ThriftSecurityException, TException {
     while (true) {
       MasterClientService.Iface client = null;
       try {
-        client = MasterClient.getConnectionWithRetry(context);
+        client = MasterClient.getConnectionWithRetry(context, 0);
```

Comment:
Could have a method `getConnectionWithRetry(ctx)` without the count that default to 0/infinite retires.  Then all of these calls do not need to pass zero.